base/changes.txt

```diff
@@ -6,6 +6,10 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2023-03-28  David Carlisle  <David.Carlisle@latex-project.org>
+
+	* ltfinal.dtx: Restore groups to \Make...case, for gh/1021
+
```

Comment:
plese use the full command names, people search for it if something goes wrong and will not find \Make...case

Reply:
done. I'll wait for gh to pick up new l3build before merging to check test suite

Reply:
> I'll wait for gh to pick up new l3build before merging to check test suite

sorry for the regression I contributed to `l3build` 😢

Reply:
happens, don't worry too much.